overview/saml.mdx

```diff
@@ -15,6 +15,45 @@ which users have access to your product. They could give everyone at the company
 access, specify teams like only the engineering team, or even pick specific
 people to give access.
 
+### SAML Setup Guide
+
+
+
+1. In the Organization Settings for your project, make sure that SAML is enabled and that all orgs can setup SAML.
+
+![organization settings](/images/saml-setup-1.png)
+
+2. Under Users, make sure that there is at least one test user with Admin or Owner priviledges.
+
+![test user](/images/saml-setup-2.png)
+
+3. On the dashboard for your project, click the "View Org Page" button under the Test Environment.
+
+![dashboard](/images/saml-setup-3.png)
+
+4. Click the Create Organization button.
+
+![create organization](/images/saml-setup-4.png)
+
+5. Write a name for the organization, and click Create Organization.
+
+![organization game](/images/saml-setup-5.png)
+
+6. On the page for the new Organization, click the Settings button, and click Enable SAML.
+
+![enable saml](/images/saml-setup-6.png)
+
+7. Click the option for the SAML Identity Provider you would like to use.
+
+![choose identity provider](/images/saml-setup-7.png)
+
+8. Follow the set by step instructions we have provided for the Identity Provider of choice.
+
+![follow IDP instructions](/images/saml-setup-8.png)
+
+
```

Comment:
I'd just add a few lines to wrap it up like As an employee of Acme Co, I was given step by step instructions on how to connect my IDP. I can then manage my employees access to your product easily. As the implementor, all you have to do is enable SAML for their organization, and PropelAuth takes care of the rest.